### PR TITLE
Replace Guggenheim with Reveal

### PR DESCRIPTION
**Pull request by justinjams**
I had trouble with Reveal trying to find a home slide using `document.querySelector('.reveal .slides>section:first-child')`, which would fail to find anything and then everything goes haywire. I modified reveal.min.js to not include the `:first-child` bit, and that works.

I'm also seeing quite a few `Invalid CSS property name:` messages in the log. I'm guessing these are properties that aren't available in `node-webkit`. Everything seems unstyled but functional, regardless.

Lastly... when it gets to the end, it loops back. I mean, like the top of a typewriter. If you cycle through the games in one direction, it will loop (the loop option is enabled) but it visually jumps backward once to get to the beginning. 

But all in all, this works fine on my machine.

Addresses #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,43 +36,31 @@ Build Instructions
 3. To test the launcher, switch to the `source` directory and run node-webkit.
 
 ```
-Linux: 
 $ cd source
 $ ../nw .
```
Comment (mildmojo) [inline README.md:41]: Actually, this is even easier: `nw source`

That means in Windows you can actually drag the source folder onto `nw.exe` to test it.

File: README.md [hunk 2]
```diff
@@ -36,43 +36,31 @@ Build Instructions
...
 > cd ..
 > copy /b nw.exe+moonshot.nw moonshot.exe
-
-OS X:
-$ cd source
-$ zip -r ../moonshot.nw *
-$ cd ..
-$ cat node-webkit.app/Contents/MacOS/node-webkit moonshot.nw > moonshot
-$ chmod u+x moonshot
-
```
Comment (mildmojo) [inline README.md:75]: No more OS X instructions? Merge error?

**Comment by justinjams** [comment]
You're right! Good catch. That's fixed, but those build steps don't exactly work for me in OS X. `moonshot.nw` executes with a double click, but `moonshot` throws:

```
dyld: Library not loaded: @executable_path/../Frameworks/node-webkit Framework.framework/node-webkit Framework
  Referenced from: /Users/justinhall/lexitron/moonshot/moonshot
  Reason: image not found
```

I think this might have to do with the way the OS X package is structured. Let me know if you have any ideas. I tried running `moonshot` from where the `node-webkit` executable resides in `node-webkit.app`, but that launches a blank `node-webkit` instance with the ASCII logo. 

I'll open an issue for it if I'm not able to figure this out shortly!

**Comment by mildmojo** [comment]
Check out the node-webkit [instructions for making an OS X executable](https://github.com/rogerwang/node-webkit/wiki/How-to-package-and-distribute-your-apps#mac-os-x). Looks like you name it app.nw and toss it into the .app.

**Comment by mildmojo** [comment]
I'd really like to find out why that reveal.js selector wasn't working rather than patching the minified library. That is, I'd like to use the markup it's trying to find, assuming webkit supports the `:first-child` pseudo-class.

**Comment by mildmojo** [comment]
Okay, so you're using Underscore templates inside the `div.slides` (nifty, haven't used those before). Could this be a timing issue? That reveal.js is looking at the DOM before Underscore has replaced the `<script>` element? Or maybe it's finding the `<script>` element instead of the `<section>` elements? Maybe the template can wrap the divs, too, making the template-generated sections direct descendants?

**Comment by justinjams** [comment]
I found that `:first-child` works if you remove the `>section` part, so it's probably not that.

I don't think it's the timing because I tested from the console I can run that query after Underscore does it's magic. 

I don't know how it could be a problem with the DOM structure, since it's based on the simplest of examples.

I'll give a shot to your last suggestion later. Good idea!

**Comment by justinjams** [comment]
It looks like the issue was caused by the first child actually being the `<script>` template. Switching the jQuery `append` of the template to a `prepend` fixed it right up.

Sorry about clobbering this branch with other commits. I'm still getting used to my workflow!

File: build.sh [hunk 3]
```diff
@@ -1,20 +1,19 @@
 #!/bin/bash
 if [ "$(uname)" == "Darwin" ]; then
-				cd source
-				zip -r ../moonshot.nw *
-				cd ..
-				cat node-webkit.app/Contents/MacOS/node-webkit moonshot.nw > moonshot
-				chmod u+x moonshot        
+	rm -rf node-webkit.app/Contents/Resources/app.nw
+	cd source
+	cp -R ./ ../node-webkit.app/Contents/Resources/app.nw
```
Comment (mildmojo) [inline build.sh:5]: I _think_ you may need a trailing slash on app.nw. Does this work okay?

File: README.md [hunk 4]
```diff
@@ -49,31 +49,27 @@ $ cd source
...
-$ cd source
-$ zip -r ../moonshot.nw *
-$ cd ..
-$ cat node-webkit.app/Contents/MacOS/node-webkit moonshot.nw > moonshot
-$ chmod u+x moonshot
+Or, for OS X, copy the files to node-webkit.app/Contents/Resources/app.nw
 
 ```
+$ cd source
+$ cp -R ./ ../node-webkit.app/Contents/Resources/app.nw
```
Comment (mildmojo) [inline README.md:71]: (same note as below)

File: source/package.json [hunk 5]
```diff
@@ -8,10 +8,10 @@
     // For production, use kiosk mode.
     //"kiosk": true,
     "fullscreen": true,
-    "frame": false,
+    "frame": true,
     //"icon": "link.png",
     //"toolbar": true,
-    "toolbar": false,
+    "toolbar": true,
```
Comment (mildmojo) [inline source/package.json]: Are these changes debugging artifacts?

BTW, if you haven't found them yet, I've got a couple of keys bound to fullscreen toggle and the web inspector in [input.js](https://github.com/runjumpdev/moonshot/blob/dfa3163b/source/lib/input.js#L43).

**Comment by mildmojo** [comment]
Cool! Glad you got reveal sorted with templates. Did you undo the modifications to reveal.min.js?

**Comment by justinjams** [comment]
Yep, reveal.min.js is reverted now. Thanks for the heads up about the debugging controls. I looked right past them before!

**Comment by mildmojo** [comment]
Awesome. Similar copy commands WFM, so I'm merging this.

**Pull request merged**